lib/tests/update.rs

```diff
@@ -681,3 +681,35 @@ async fn check_permissions_auth_disabled() {
 		);
 	}
 }
+
+#[tokio::test]
+async fn update_only() -> Result<(), Error> {
+	let sql: &str = "
+		UPDATE ONLY test;
+		UPDATE ONLY test:1;
+		UPDATE ONLY test:2, test:3;
+	";
+	let dbs = new_ds().await?;
+	let ses = Session::owner().with_ns("test").with_db("test");
+	let res = &mut dbs.execute(sql, &ses, None).await?;
+	assert_eq!(res.len(), 3);
+	//
+	let tmp = res.remove(0).result?;
+	let val = Value::parse("NONE");
```

Comment:
Technically `UPDATE test` returns multiple results. I would expect `UPDATE ONLY test` to return a `SingleOnlyOutput` error here.

Reply:
In this case, no records on the `test` table exist yet, which is why it results in `[]`, thus returning `NONE` with the `ONLY` keyword

Reply:
I understand that but that makes it non-deterministic. A query like that will start returning an error as soon as there are at least two results in that table. Something like `UPDATE ONLY test:1`, on the other hand, will always return no results or one result only.

Reply:
`NONE` because email is unique so that query will always return at most one result.

Reply:
Yes, but if no record for that email exists then the update statement will result in `[]`, thus `NONE`, just like the example above. I don't want to make a different between updating a table and applying `WHERE` filters as that will cause confusion.

Could you sketch an example query that explains the problem you foresee?